client/state/comments/README.md

```diff
@@ -12,8 +12,10 @@ comments = {
 
 ## Types
 
+```text
```

Comment:
Should these be list items? `* foo` or something?

Reply:
I think that could work as well. I just added the `text` mark to minimize the change, but maybe in this case it would be better to convert a "fake" codeblock in a list. No strong opinions either way.